src/audio/codec_adapter/codec/cadence.c

```diff
@@ -681,16 +680,6 @@ cadence_codec_set_configuration(struct processing_module *mod, uint32_t config_i
 	    md->state < MODULE_INITIALIZED)
 		return 0;
 
-	/* whole configuration received, apply it now */
```

Comment:
@dbaluta if we dont apply the config right away here, the check just above is redundant right?

Reply:
@ranj063 true.